backend/core/src/auth/services/sms-mfa.service.ts

```diff
@@ -11,7 +11,7 @@ export class SmsMfaService {
   ) {}
   public async sendMfaCode(user: User, phoneNumber: string, mfaCode: string) {
     return await this.client.messages.create({
-      body: `Your bloom account access token: ${mfaCode}`,
+      body: `Your Partners Portal account access token: ${mfaCode}`,
```

Comment:
Based on the email below it looks like these are meant to be translated strings - if this issue is on a timeline I'm good to make a separate ticket